workflow/training/classification.py

```
@@ -10,8 +10,9 @@
 from functools import partial
 import numpy as np
 from sklearn.preprocessing import LabelEncoder
-
-
+from huggingface_hub import HfApi, CommitOperationAdd
+import io 
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)